docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_creating_override_snapshot.adoc

```diff
@@ -0,0 +1,70 @@
+= Creating an override snapshot
+
+In {ProductName}, integration service provides the users with the ability to reset their component's Global Candidate List to a desired state with a manually created `override` snapshot which can then be a candidate for further promotion (release) if it passes all integration tests subsequently. 
```

Comment:
"In {ProductName}, the integration services provides users..." 

I would also suggest removing "subsequently" from the end of this line. You don't need it, and it will make the sentence slightly shorter and easier on readers.